main.py

```diff
@@ -2,6 +2,42 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson"),
+    ]
+
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    for actor in actors:
+        Actor.objects.create(first_name=actor[0], last_name=actor[1])
```

Comment:
unpack tuple here

![image](https://github.com/mate-academy/py-genres-and-actors/assets/108459229/7a14f9d5-5722-4fea-8925-a79923990323)